engine/src/main/java/org/hibernate/search/engine/reporting/spi/RootFailureCollector.java

```diff
@@ -230,12 +229,11 @@ private void doAdd(Throwable failure, String failureMessage) {
 			appendContextTo( contextJoiner );
 			log.newCollectedFailure( root.process, contextJoiner.toString(), failure );
```

Comment:
I wonder, now that this log has become a TRACE, is it worth rendering the context if the log level is not TRACE?

Reply:
Right... I'll see about that.